boskos/boskos.go

```diff
@@ -36,6 +36,7 @@ import (
 var (
 	configPath        = flag.String("config", "config.yaml", "Path to init resource file")
 	storagePath       = flag.String("storage", "", "Path to persistent volume to load the state")
+	heartBeat         = flag.Int("heartbeat", 10, "Duration (in minutes) after which boskos updates all the resources.")
```

Comment:
hummm this is when boskos reloads the config, not the resource heartbeat I think

Reply:
reverted. should be handled by boskos client user e.g kubetest.